packages/app/src/app/pages/Dashboard/Sidebar/SandboxesItem/FolderEntry/index.tsx

```diff
@@ -54,6 +55,9 @@ type Props = {
   currentPath: string;
   currentTeamId: string;
 
+  // router props
```

Comment:
We can add these props by doing

```
type Props = { .. } & RouteComponentProps;
```

Then we have it properly typed as well 😄

Reply:
Hey, thanks for the quick help.  But this doesn't seem to work. Any ideas why?

Reply:
I'll take a look!